src/pydata_sphinx_theme/assets/styles/components/_navbar-links.scss

```diff
@@ -1,16 +1,10 @@
 /**
  * Navigation links in the navbar and icon links
  */
-.navbar-nav,
-.navbar-icon-links {
+ul.navbar-nav {
```

Comment:
I decided to add `ul` to the front of this selector to re-enforce the idea that the Bootstrap docs show this class is generally put on a `ul` and that's the way we should use it too.